index.js

```diff
@@ -333,9 +333,9 @@ app.post('/api/reactions/:id', cors(corsOptions), async (req, res) => { // react
 
 app.get('/auth/begin', (req, res) => {
     if (req.get('host') == 'localhost:8081') {
-        res.redirect(`https://fluffyscratch.hampton.pw/auth/getKeys/v2?redirect=bG9jYWxob3N0OjgwODEvYXV0aC9oYW5kbGU=`)
+        res.redirect(`https://auth.itinerary.eu.org/auth/?redirect=bG9jYWxob3N0OjgwODEvYXV0aC9oYW5kbGU=&name=Ocular`)
     } else {
-        res.redirect(`https://fluffyscratch.hampton.pw/auth/getKeys/v2?redirect=bXktb2N1bGFyLmplZmZhbG8ubmV0L2F1dGgvaGFuZGxl`)
+        res.redirect(`https://auth.itinerary.eu.org/auth/?redirect=aHR0cDovL2xvY2FsaG9zdDo4MDgxL2F1dGgvaGFuZGxl&name=Ocular`)
```

Comment:
```suggestion
        res.redirect(`https://auth.itinerary.eu.org/auth/?redirect=bXktb2N1bGFyLmplZmZhbG8ubmV0L2F1dGgvaGFuZGxl&name=Ocular`)
```
Fix redirect location